### PR TITLE
fix a typo

### PR DESCRIPTION
**Pull request by RiverAge**
it should be weixin not qq.

### CODE REVIEW

File: src/Adapters/Auth/wechat.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ function validateAuthData(authData) {
     if (data.errcode == 0) {
       return;
     }
-    throw new Parse.Error(Parse.Error.OBJECT_NOT_FOUND, 'qq auth is invalid for this user.');
+    throw new Parse.Error(Parse.Error.OBJECT_NOT_FOUND, 'weixin auth is invalid for this user.');
```
Comment (flovilmart) [inline src/Adapters/Auth/wechat.js]: Or wechat?

Comment (RiverAge) [inline src/Adapters/Auth/wechat.js]: weixin is usually used in Chinese.
Yes,wechat is better.

**Comment by RiverAge** [comment]
@flovilmart I think the second check of response is unnecessary!

**Comment by flovilmart** [comment]
Yes! Very useless :)

**Pull request merged**